### DISCUSSION TITLE
Prioritise a specific monster in a stack

### DISCUSSION THREAD

**Discussion by NMOSRS** [runelite/runelite#15141] [Ideas]
It would be great to be able to prioritise a specific monster in a stack of monsters. This would effectively allow you to always left-click a monster despite multiple other monsters being on top of them.

e.g. when you are killing Commander Zilyana, Starlight will occasionally walk under her making Zilyana harder to left-click. With this suggestion Zilyana will always have left-click priority.

**Comment by geheur** [comment]
You should already be able to do something similar by setting the left-click walk here swap on Starlight. You'll have to right-click starlight when you want to kill it though.

**Reply by NMOSRS** [reply]
I was concerned that if left-click walk here was enabled on Starlight and her model was over Zilyana's that I would just walk to where she was instead of clicking Zilyana. Haven't been able to test just yet

**Reply by geheur** [reply]
My testing with the crafting guild cows tells me that this isn't the case. The code does not actually swap "Walk here", rather it swaps all the NPC's options below "Walk here".

**Reply by NMOSRS** [reply]
Ah, that's a perfect work-a-round in that case! Thanks for this
